# goal definition
std_msgs/Header header
int64[] apriltag_ids
---

# result: list of cubes' positions
std_msgs/Header header
geometry_msgs/Point[] positions
---
# feedback
std_msgs/Header header
string robot_status
